### ISSUE TITLE
[Feature Request] hldm force enemy model

### ISSUE THREAD

**Issue by crustypig** [ValveSoftware/halflife#722]
It would be great if we could get a cvar which forced the enemy model and team mate models to models of your choice, similar to what they do in quake, as some of the models clash with background textures and can hurt your eyes. for example on some maps it can be an unfair advantage if your opponent plays as scientist with awkward colours because the models arent replaceable so people use it to troll you

just a suggestion

**Comment by 0Ky** [comment]
I agree with @crustypig https://github.com/ValveSoftware/halflife/issues/722#issue-11694356. In the 25th anniversary update for Half-Life, there were several new added models, one of which is the "skeleton" model. This model in particular makes it challenging for opponents to perceive and accurately target, which gives the player using this model a tactical advantage.

<details>
  <summary>Visual Demonstration</summary>

___
![gordon](https://github.com/ValveSoftware/halflife/assets/16103757/5e6cba0e-08e0-4ccd-9a7d-3d72c287111d) ![skeleton](https://github.com/ValveSoftware/halflife/assets/16103757/0e4150c3-e0bb-4127-99ed-8bfdf2a8bd1c)
___
</details>

In Counter-Strike 1.6 there is a `cl_minmodels` cvar, when enabled, it simplifies player model rendering by limiting model variety to a minimum, using `cl_min_ct` and `cl_min_t` to determine specific model index for CT and T teams, respectively. It would be great if Half-Life also had implemented a similar command to force all enemy/team players to a specific default model by index value. This feature aims to reduce visual clutter and facilitate easier identification of players.

I've also come across a community server, which had custom models that made the enemy team entirely invisible...
